DOCS-2401: (BAST-1546) Adding information about Delete and Dismiss scan results (#1774)
tigera/docs: calico-cloud/image-assurance/understanding-scan-results.mdx

## All Scanned Images tab

This tab lists scanned images if you have enabled or used one of the [Image Assurance scanners](./scanners/overview).
To manage your scan results, you can filter, dismiss, or delete them:
* **Filter:** Apply different combinations of filters to refine your results, making it easier to focus on relevant vulnerabilities for remediation.

> * **Dismiss:** Temporarily hide scan results that are not immediately important. Dismissed scan results will be marked as 'Unknown', and you can review or restore them by enabling the Dismissed filter and turning it on.

Reviewer: Suggestion:

`Dismiss: Hide specific results from the scan results list. You can view a list of dismissed scan results by using the **Dismissed** filter.`

> * **Delete:** Permanently remove scan results that are no longer relevant to your remediation efforts.

Reviewer: ```suggestion
* **Delete:** Permanently remove specific results from the scan results list.
```

## All Scanned Images tab

This tab lists scanned images if you have enabled or used one of the [Image Assurance scanners](./scanners/overview).

#### Filter Scan Results

You can use different combinations of filters to refine your results, making it easier to monitor and focus on relevant vulnerabilities for your remediation efforts.

#### Delete Scan Results

If a scan result is no longer relevant to your remediation effort, you can delete it by selecting multiple results and choosing the **Delete** option from the Bulk Actions dropdown.

:::note

**Deleted Results**: Deleting a scan result removes it from the **All Scanned Images** tab. However, if the image is currently running, its record will still appear in the **Running Images** tab, with the number of instances tracked.

> :::

Reviewer: Do we really need this? I'm not sure we'd expect deleting a result to mean ALL results are removed for that image.

Author: Agree. This can be omitted as users likely expect this behavior.

#### Dismiss/Restore Scan Results Temporarily

If you want to temporarily hide scan results that are not important at the moment (e.g., "noise" or less critical vulnerabilities), you can dismiss them by selecting the rows and choosing the **Dismiss** option from the Bulk Actions dropdown.

Dismissed results can be accessed at any time for future review by enabling the **Dismissed** toggle filter. To restore them you can select the rows and choose the **Restore** option from the Bulk Actions dropdown.

:::note

**Dismissed Results**: While a scan result is dismissed, it will have no impact on the Admission Controller policies.

> :::

Reviewer: Ambiguous. What do you mean?
* Even if you dismiss a scan result, that result will continue to affect the Admission Controller policies.
* If you dismiss a scan result, the Admission Controller policies will no longer be affected by that scan result.

Author: "If you dismiss a scan result, the Admission Controller policies will no longer be affected by that scan result."
But this can be omitted as users likely expect this behavior.
